tools/runners/VeribleExtractor.py

```diff
@@ -33,16 +33,27 @@ def prepare_run_cb(self, tmp_dir, params):
         inc_dirs = ",".join(params.get("incdirs", []))
 
         with open(script_path, "w") as script:
-            s = (
-                'log="$({executable}'
+            command = (
+                '{executable}'
                 ' --file_list_root ""'
                 ' --include_dir_paths {inc_dirs}'
-                ' --file_list_path {src_list_path}'
-                ' 2>&1 1>/dev/null)"\n'
-                'if [ -n "$log" ]; then echo "$log"; exit 1; fi\n').format(
+                ' --file_list_path {src_list_path}').format(
                     executable=self.executable,
                     inc_dirs=shlex.quote(inc_dirs),
                     src_list_path=shlex.quote(src_list_path))
+            s = (
```

Comment:
Maybe add a comment that describes what is happening  - that we use the log output as a proxy information that something failed unless it is talking about re-defining a macro.

Reply:
Fixed